### PR TITLE
[TASK] Add link to "How to use DI in XCLASSed class" blog article

### PR DESCRIPTION
**Pull request by brotkrueml**
This question comes up sometimes in Slack, so we can also reference to a solution.

Releases: main, 12.4, 11.5

### CODE REVIEW

File: Documentation/ApiOverview/Xclasses/Index.rst [hunk 1]
```diff
@@ -134,3 +134,6 @@ The object-oriented rules of PHP, such as rules about visibility, apply here.
 As you are extending the original class you can overload or call methods
 marked as public and protected but not private or static ones. Read more about
 `visibility and inheritance at php.net <https://www.php.net/manual/en/language.oop5.visibility.php>`__
+
+..  seealso::
+    `How to use constructor dependency injection in a XCLASSed TYPO3 class <https://www.derhansen.de/2021/06/how-to-use-constructor-injection-with-typo3-xclass.html>`__
```
Comment (linawolf) [inline Documentation/ApiOverview/Xclasses/Index.rst:139]:
```suggestion
    `How to use constructor dependency injection in a XCLASSed TYPO3 class 
        <https://www.derhansen.de/2021/06/how-to-use-constructor-injection-with-typo3-xclass.html>`__
```

Comment (brotkrueml) [reply Documentation/ApiOverview/Xclasses/Index.rst]: What is the difference?

Comment (linawolf) [reply Documentation/ApiOverview/Xclasses/Index.rst]: We strive to have a maximal  line length of 80

Comment (brotkrueml) [reply Documentation/ApiOverview/Xclasses/Index.rst]: Yup, but this is a soft convention. And PhpStorm does not recognize them anymore correctly with a line break - why I would not break the line anymore.

**Pull request merged**